Jenkinsfile

```diff
@@ -128,6 +128,8 @@ pipeline {
                                 IS_ES_IMAGE_UPDATED = 'false'
```

Comment:
In future ES bump PRs, if we need to force an image build and push, temporarily overriding this variable to `true` will do that.